packages/carousel/src/Carousel.js

```diff
@@ -88,7 +88,7 @@ class Carousel extends React.Component {
       width: PropTypes.number,
       height: PropTypes.number,
       margin: PropTypes.number,
-      fontSize: PropTypes.string,
+      fontSize: PropTypes.number,
```

Comment:
this is a breaking change, please inform this on the commit message

Reply:
I'll make a rebase on the commit